browser/main/SideNav/StorageItem.js

```diff
@@ -131,8 +134,39 @@ class StorageItem extends React.Component {
     }
   }
 
+  handleDragStart (e, folder) {
+    e.dataTransfer.setData('dragFolderKey', folder.key)
+  }
+
+  handleDrop (e, folder, storage, dispatch) {
+    const dragFolderKey = e.dataTransfer.getData('dragFolderKey')
+    this.changeOrderFolder(dragFolderKey, folder.key, storage.folders)
+    CSON.writeFileSync(path.join(storage.path, 'boostnote.json'), _.pick(storage, ['folders', 'version']))
+    dispatch({
+      type: 'UPDATE_FOLDER',
+      storage: storage
+    })
+  }
+
+  changeOrderFolder (dragFolderKey, targetFolderKey, folders) {
+    const dragFolderIndex = this.getFolderIndex(folders, dragFolderKey)
+    const targetFolderIndex = this.getFolderIndex(folders, targetFolderKey)
+    this.moveFolderElements(folders, dragFolderIndex, targetFolderIndex)
+  }
+
+  getFolderIndex (folders, targetFolderKey) {
+    const folderIndex = folders.indexOf(_.find(folders, {key: targetFolderKey}))
+    return folderIndex
```

Comment:
You can more simplify like `return folders.indexOf(_.find(folders, {key: targetFolderKey}))`